cmd/entrypoint/watcher.go

```diff
@@ -33,6 +35,16 @@ func WatchAllTheThings(
 	if err != nil {
 		return err
 	}
+	intv, err := strconv.Atoi(env("AMBASSADOR_RECONFIG_MAX_DELAY", "1"))
```

Comment:
So, why was 1 second introduced? I thought the default was 0 to maintain existing behavior of sending each update as they are received?

Reply:
It's mostly because the time.Ticker accepts only a non-zero positive interval and panics otherwise. So if we wanted to keep/support 0 interval you would have to hard code some interval between 0 and 1 seconds but then its not really 0 is it? 

From load tests, it seems with a slightly above 0 interval (e.g. 100ms) interval it would do a bit more reconfigurations that what it was previously so I chose the lowest non-zero second instead.